process/README.md

```diff
@@ -22,8 +22,16 @@ flowchart TD
 The process is designed to be flexible to enable a Project to move in and out of a Working Group as deemed appropriate by the TAC.
 
 # II. Lifecycle
+<img align="top" src="https://github.com/ossf/tac/blob/main/process/TI-lifecycle-stages.png">
```

Comment:
Breaking up walls of text with images is nice, but do we want some sort of lead-in here like "Here are the four stages of the TI lifecycle:"?

Reply:
I actually don't think that this image really adds anything as it stands, it should at least have some arrows showing the progression from one stage to the other. Otherwise it's just duplicative of the bulleted list.

Reply:
I'll add some arrows to illustrate workflow, thanks for the suggestion

Reply:
I actually replaced the image with a mermaid graph that has the arrows.